lib/matplotlib/text.py

```diff
@@ -1657,17 +1657,41 @@ def set_figure(self, fig):
 
 
 class OffsetFrom(object):
+    '''Callable helper class for working with `Annotation`
```

Comment:
Raymond Hettenger in https://www.youtube.com/watch?v=wf-BqAjZb8M suggest
triple single quotes for docstrings.

On Sat, May 28, 2016 at 4:40 PM Elliott Sales de Andrade <
notifications@github.com> wrote:

> In lib/matplotlib/text.py
> <https://github.com/matplotlib/matplotlib/pull/6464#discussion_r64994068>:
>
> > @@ -1657,17 +1657,41 @@ def set_figure(self, fig):
> >
> >
> >  class OffsetFrom(object):
> > +    '''Callable helper class for working with `Annotation`
>
> I'm normally a proponent of single quotes because they're easier to type,
> but isn't the convention to use double quotes for docstrings? Not that
> that's a huge difference or anything...
>
> —
> You are receiving this because you authored the thread.
>
> Reply to this email directly, view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6464/files/bf710e0e353364ff0328d515a200fb2a11f859f8#r64994068>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAMMhUVxntJ7GX4t2ruF4isYAqhh2bDSks5qGKgrgaJpZM4IkE5->
> .
>